polynote-frontend/webpack.config.js

```diff
@@ -6,11 +6,12 @@ const path = require('path');
 
 module.exports = {
   entry: './polynote/main.ts',
-  devtool: 'source-map',
+  devtool: 'eval-cheap-module-source-map',
```

Comment:
Do these changes belong in this PR?

Reply:
No, they're from https://github.com/polynote/polynote/pull/1030 (the PR is a merge to that branch so I'm not sure why it's visible. Anyways, I'll rebase this branch onto master.

Reply:
Done!